### PR TITLE
Week One Assignment

### PR DESCRIPTION
**Pull request by edrischhawniwala**

### CODE REVIEW

File: Week-1/index.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
+<html>
+    <head>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <style>
```
Comment (avinashrock) [inline Week-1/index.html]: Instead of adding css here itself , try to write css in a separate file and then import here

File: Week-1/index.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+            .playlist-songsbar .playlist-image {
+                padding: 10px;
+            }
+            .playlist-songsbar .playlist-image img {
+                height: 100px;
+                width: 100px;
+            }
+        </style>
+    </head>
+    <body>
```
Comment (avinashrock) [inline Week-1/index.html:7]: Try to use some semantic html

**Review by avinashrock — COMMENTED** [review]
Added minor comments

File: Week-1/index.html (hunk 2 above)
Comment (edrischhawniwala) [reply Week-1/index.html:7]: Hi @avinashrock,

I have added semantic html tags as requested.

File: Week-1/index.html (hunk 1 above)
Comment (edrischhawniwala) [reply Week-1/index.html]: I have created a separate stylesheet and moved all the css into style.css file.

**Review by avinashrock — APPROVED** [review]

**Pull request merged**